tutorials/scripting/gdscript/gdscript_basics.rst

```diff
@@ -264,7 +264,12 @@ The following is the list of supported operators and their precedence.
 +--------------------------------------------------------------------------------+-------------------------------------------+
 | ``<`` ``>`` ``==`` ``!=`` ``>=`` ``<=``                                        | Comparisons                               |
 +--------------------------------------------------------------------------------+-------------------------------------------+
-| ``in``                                                                         | Content test                              |
+| ``in``                                                                         | When used with the 'if' keyword it checks |
+|                                                                                | if a value is within a string, list,      |
+|                                                                                | range, dictionary, or node.               |
+|                                                                                | When used with the 'for' keyword it is    |
+|                                                                                | used to iterate though the contents of a  |
+|                                                                                | string, list, range, dictionary or node.  |
```

Comment:
Please amend this part to use the reStructuredText syntax for inline code blocks (surround "if" and "for" with **two** backticks, and do not put quotes).

Reply:
Syntax fix applied.